modules/core/src/main/scala/ch/epfl/bluebrain/nexus/admin/core/resources/Resources.scala

```diff
@@ -101,17 +105,15 @@ class Resources[F[_], A: IdResolvable: TypeFilterExpr](agg: Agg[F], sparqlClient
     * @param id     the identifier of the resource
     * @param rev    the last known revision of the resource instance
     * @param value  the json payload of the resource
-    * @param tags   the tags added to the consequent [[ResourceEvent]] which might be created as a result of this operation
-    * @param persId the unique persistence ID of the resource
     * @return a [[RefVersioned]] instance wrapped in the abstract ''F[_]'' type
     *         if successful, or a [[ch.epfl.bluebrain.nexus.admin.core.Fault]] wrapped within ''F[_]'' otherwise
     */
-  def update(id: A, rev: Long, value: Json)(tags: Set[String], persId: String)(
-      implicit ctx: CallerCtx,
-      @silent perms: HasWriteProjects): F[RefVersioned[A]] =
+  def update(id: A, rev: Long, value: Json)(implicit ctx: CallerCtx): F[RefVersioned[A]] =
```

Comment:
you need an evidence that you can call that method, otherwise anybody can.
`@silent perms: HasWriteProjects `